src/nvim/edit.c

```diff
@@ -8235,9 +8231,8 @@ static int ins_eol(int c)
     curwin->w_cursor.col += (colnr_T)STRLEN(get_cursor_pos_ptr());
 
   AppendToRedobuff(NL_STR);
-  i = open_line(FORWARD,
-      has_format_option(FO_RET_COMS) ? OPENLINE_DO_COM :
-      0, old_indent);
+  int i = open_line(FORWARD, has_format_option(FO_RET_COMS)
```

Comment:
I would suggest move `has_format_option` to its own line to make ?: fit on the same. Also `i` is `bool`.